Sources/SPTDataLoader.m

```diff
@@ -222,9 +268,17 @@ - (void)receivedDataChunk:(NSData *)data forResponse:(SPTDataLoaderResponse *)re
     
     BOOL didReceiveDataChunkSelectorExists = [self.delegate respondsToSelector:@selector(dataLoader:didReceiveDataChunk:forResponse:)];
     if (didReceiveDataChunkSelectorExists) {
-        [self executeDelegateBlock: ^{
+        [self executeDelegateBlock:^{
             [self.delegate dataLoader:self didReceiveDataChunk:data forResponse:response];
         }];
+    } else {
+        [self executeDelegateBlock:^{
+            void (^handler)(SPTDataLoaderResult *) = [self.requestHandlers objectForKey:response.request];
+            handler([SPTDataLoaderResult fromResponse:response withChunk:data]);
```

Comment:
Always check for `nil` before invoking a block. If `handler == nil`, this will crash.